src/query/__tests__/RelayQueryRoot-test.js

```diff
@@ -342,6 +342,77 @@ describe('RelayQueryRoot', () => {
     expect(meIdentifyingArg).toBeUndefined();
   });
 
+  it('returns the identifying number argument ', () => {
+    var nQuery = getNode(Relay.QL`
+      query {
+        task(number:5){
+          dumb,
+        }
+      }
+    `);
+    nQuery.getConcreteQueryNode().metadata= {
+      identifyingArgName: 'number',
+      identifyingArgType: 'scalar' };
```

Comment:
That's cool, just checking. If the test passes without explicitly setting the metadata (`.metadata = ...`), then let's skip setting it. That way we're testing whether a query as written by a user will produce the correct runtime behavior.